app/Http/Services/Tabbar/TabbarService.php

```diff
@@ -92,14 +99,15 @@ private function _getContentsTabInfo($contentsParam, $contentsTabUrlInfo) {
     }
 
     /**
-     * Devタブの情報を取得する
+     * Devタブの情報を取得する.
      * 
-     * @param string $devParam リクエストパラメータのDevの値
-     * @param array $devTabUrlInfo Devタブ内に表示するページのURL一覧
+     * @param string $devParam      リクエストパラメータのDevの値
+     * @param array  $devTabUrlInfo Devタブ内に表示するページのURL一覧
+     *
      * @return string Devタブに表示するページのURL
-     * 
      */
-    private function _getDevTabInfo($devParam, $devTabUrlInfo) {
+    private function _getDevTabInfo($devParam, $devTabUrlInfo)
```

Comment:
__[code_sniffer]__
* Method name "_getDevTabInfo" should not be prefixed with an underscore to indicate visibility